### PR TITLE
Present() methods changed from LM_F-->LM_T

### PR DESCRIPTION
**Pull request by raconte**
Implements #1398

Has been modified to include a tracelevel global for all the cases changed "LmtPresent" and changed from LM_F to LM_T.

### CODE REVIEW

**Comment by raconte** [comment]
Fixed all in bf5ef7f and b8159d5

File: archive/proxyCoap/src/app/proxyCoap.cpp [hunk 1]
```diff
@@ -92,7 +92,7 @@ PaArgument paArgs[] =
 */
 void sigHandler(int sigNo)
 {
-  LM_F(("In sigHandler - caught signal %d", sigNo));
+  LM_F(LmtPresent, ("In sigHandler - caught signal %d", sigNo));
```
Comment (kzangeli) [inline archive/proxyCoap/src/app/proxyCoap.cpp]: LM_T

**Comment by kzangeli** [comment]
origin/develop
Now, what's missing here is a merge with develop, and ma "make test", to make sure nothing is broken

File: archive/proxyCoap/src/app/proxyCoap.cpp (hunk 1 above)
Comment (raconte) [inline archive/proxyCoap/src/app/proxyCoap.cpp]: Fixed in d2e28c9

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**